### DISCUSSION TITLE
adapt column sizes in baseof.html

### DISCUSSION THREAD

**Discussion by huehnerlady** [google/docsy#644] [Q&A] [answered]
Hi,

I am currently creating a new website for our charity using docsy. This is not a technical documentation I know, but as I like the feel of docsy I wanted to give it a go :) 

So far it works quite well. Now I run into the problem, that I would like to make the main part in `baseof.html` a bit smaller and the right sidebar with the toc a bit wider.

I found out that the classes `col-xl-<1, 2, 3....>` are part of it, but as soon as I try to use a different size or try to override it with my own class, the page ends up being wider than 100%, even though the percentages are 25/25/50. 

How can I adapt the column sizes to my need?
I could not find anything in the documentation either and I assume it is because it is quite an edge case to change the width. But maybe someone did that and can help here?

**Comment by narrenfrei** [comment, marked as answer]
The definition of the widths is partly deeply integrated with Bootstramp and depends also on the screen size.

Per default Docsy is using a grid with 12 columns (I think). So you have to be aware that the sum of defined column-widths isn't above 12.

Maybe this Docs will help you: https://getbootstrap.com/docs/4.0/layout/grid/#grid-options

**Reply by huehnerlady** [reply]
@narrenfrei thanks, that helps a lot. I did keep that in mind while trying again and that works as wanted, so many thanks
